### PR TITLE
feat: add game page issues & round section

### PR DESCRIPTION
**Pull request by utrennik**
Capture

*** Game page ISSUES/ROUND section ***
* should contain issues list
* selected issue should be highlighted
* played issues should be highlighted (another way than selected, i propose for exanple the green line on the left)
* for the selected via click Issue should be displayed: full title, description and score
* score should contain the average score of the issue and statistics (cards values select percentage)
* should contain the Run round / Restart round button (depending on was the Issue played or not)
* should contain flip cards button (for DEALER manual round end)
* should contain add issue card
* should contain the draft data

Out of scope:
* Select issue triggers only the details and score display
* run / reset round buttons are doing nothing

### CODE REVIEW

File: client/src/components/pages/game-page/game-round-section/stats-item/stats-item.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { IStatsItem } from '@models/types';
+import config from '@src/config.json';
+import './stats-item.sass';
+
+export const StatsItem = ({ score, percentage, pointsShortName }: IStatsItem) => (
+  <div className="stats-item">
+    <div className="stats-item-value">
+      {(score === config.PASS && 'PASS') ||
+        (score === config.COFFEE && '?') ||
+        (score && `${score} ${pointsShortName}`)}
```
Comment (aksiantsiuk) [inline client/src/components/pages/game-page/game-round-section/stats-item/stats-item.tsx]: Better to define variable(s) for it like `statItemValue`

File: client/src/components/ui/issueCard/issueCard.tsx [hunk 2]
```diff
@@ -45,12 +43,16 @@ export const IssueCard = ({ id, name, priority, isActive, isGame, isDealer }: II
...
-
   return (
     <div title={name}>
-      <Card className={isueCardStyles}>
+      <Card
+        className={
+          (isActive && isPlayed && 'issue-card selected played') ||
+          (isActive && 'issue-card selected') ||
+          (isPlayed && 'issue-card played') ||
+          'issue-card'
```
Comment (aksiantsiuk) [inline client/src/components/ui/issueCard/issueCard.tsx]: The same

File: client/src/models/types.ts [hunk 3]
```diff
@@ -122,6 +122,19 @@ export enum IssuePriority {
   LOW = 'low',
 }
 
+export interface IStatsItem {
+  score: string | 'pass' | 'coffee'[];
+  percentage: number;
+  pointsShortName: string;
+}
+
+export type VotingData = string | 'pass' | 'coffee';
```
Comment (aksiantsiuk) [inline client/src/models/types.ts]: Better to reduce code duplication:
```ts
enum VotingData = {
  ONE: '1' 
  PASS: 'pass',
 COFFEE: 'coffee',
 ....
}

export interface IStatsItem {
  score: keyof typeof VotingData[];
  percentage: number;
  pointsShortName: string;
}
```

**Review by HelenBassa — APPROVED** [review]

File: client/src/components/pages/game-page/game-round-section/stats-item/stats-item.tsx (hunk 1 above)
Comment (utrennik) [reply client/src/components/pages/game-page/game-round-section/stats-item/stats-item.tsx]: Done

File: client/src/components/ui/issueCard/issueCard.tsx (hunk 2 above)
Comment (utrennik) [reply client/src/components/ui/issueCard/issueCard.tsx]: Done

File: client/src/models/types.ts (hunk 3 above)
Comment (utrennik) [reply client/src/models/types.ts]: Didn't noticed) Fixed

**Pull request merged**